config/routes.rb

```diff
@@ -83,6 +83,9 @@
   direct :contact_form do
     { controller: 'welcome', action: 'show', anchor: 'help' }
   end
+  # legacy /contact links in PURLs should redirect to new contact form URL
```

Comment:
I think it would be helpful to provide an example of where this is "in PURLs".  Then we can check to see if this is still needed at some point in the future.

Reply:
@andrewjbtw do you have a couple examples we can drop in a comment here?  OR could link to the code ticket and/or to a remediation ticket for reference

Reply:
There is a purl-feedback ticket: https://jirasul.stanford.edu/jira/browse/PURL-1206

The most recent conversation started with this purl: https://purl.stanford.edu/gp739cs2671 But it's come up a few times recently, leading me to think about redirects.

Reply:
Peter so if you can say "encoded in the rightsMetadata XML", that would be helpful.

Excerpt from gp739cs2671
```xml 
<rightsMetadata>
...
<use>
  <human type="useAndReproduction">
Property rights reside with the repository. Literary rights reside with the creators of the documents or their heirs. To obtain permission to access, publish or reproduce this content, please contact the Stanford Digital Repository (http://sdr.stanford.edu/contact).
  </human>
  <license/>
</use>
</rightsMetadata>
```

Reply:
added code comments